src/components/screen-reader/screen-reader.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-screen-reader',
+  templateUrl: './screen-reader.component.html',
+  styleUrls: ['./screen-reader.component.scss'],
+})
+export class ScreenReaderComponent {
+  @Input() repetitionDelay = 500; // duration (in ms) for which empty the live text when the same text needs to be read out consequently
+
+  liveText: string | null = null;
```

Comment:
I don't think `ngOnChanges` can handle repetition (unless having a wrapper object as an `@Input`). Angular change detection would not trigger `ngOnChanges` in case of repetition, because `'text to be repeated' === 'text to be repeated'`.